openstef/metrics/reporter.py

```diff
@@ -126,19 +126,22 @@ def get_metrics(y_pred: np.array, y_true: np.array) -> dict:
         return results
 
     @staticmethod
-    def write_report_to_disk(report: Report, location: Path):
-        """Write report to disk,
-        easy for e.g. viewing report of latest models using grafana"""
-        # create path if does not exist
-        if not os.path.exists(location):
-            os.makedirs(location)
-        # write feature importance figure
-        report.feature_importance_figure.write_html(f"{location}/weight_plot.html")
-        # write predictors
-        for name, figure in report.data_series_figures.items():
-            figure.write_html(f"{location}/{name}.html")
+    def write_report_to_disk(report: Report, artifact_folder: str):
+        """Write report to disk; e.g. for viewing report of latest models using grafana."""
+        if artifact_folder:
+            # create path if does not exist
+            if not os.path.exists(artifact_folder):
+                os.makedirs(artifact_folder)
+            # write feature importance figure
+            report.feature_importance_figure.write_html(
+                f"{artifact_folder}/weight_plot.html"
+            )
+            # write predictors
+            for name, figure in report.data_series_figures.items():
+                figure.write_html(f"{artifact_folder}/{name}.html")
```

Comment:
This should be {artifact_folder}/{pid}/{name}.html.
With this implementation, e.g. 'Predictors_47.0.html' is stored in ../trained_models/ and is overwritten by the next trained model.